docs/contexto.md

```diff
@@ -0,0 +1,44 @@
+# Contexto
+
+Como dueño de un restaurante y empresario mi objetivo es poder obtener el mayor beneficio entre lo ganado por todos los servicios a los comensales y mis costes relacionados con servirles.
+
+## Factores que influyen en el beneficio
+- Número de comensales totales a las que puedo servirles por unidad de tiempo.
+- Ingreso medio por comensal.
+
+Destacar que el número de comensales puede aumentarse permitiendo que haya más personas en el restaurante o acelerando el período que tarda un comensal desde que se sienta hasta que se va.
+Además, cuanto mejor sea la experiencia de los comensales, más estarán dispuestos a pagar.
+
+## Factores que influyen en los costes
+- Comida.
+- Número de personas necesarias para servir.
+- Número de horas por cada persona que sirve.
+- Alquiler del local o compra de éste.
+- Impuestos, auditoría y otros.
+
+## Nuestro Sistema
+
+### Beneficio
+
+Nuestro sistema permite ayudar a los dueños de los restaurantes a:
+- Servir más rápido a los comensales.
+- Servir un mayor número de comensales.
+- O reducir el personal necesario (número total u horas por personal) para dar servicio a cada comensal.
+
+Se dice "o" en el último elemento porque el dueño del restaurante será el que decida la estrategia que quiere seguir porque:
+- A mayor número de mesas, o se necesitará más personal o más tiempo se tardará por mesa.
+- A mayor personal, menos tiempo se tardará por mesa o mayor número de mesas se podrá servir.
+
+Sin embargo, este sistema le permitirá conseguir mayor ingresos independientemente de la estrategia.
+
+### ¿Cómo?
+
+Nuestro sistema ayudará a la comunicación entre el distinto personal y mejorará la toma de decisiones en el restaurante.
+
+### Suposiciones
+
+- La persona propietaria o administradora del software describe las bebidas, entrantes, platos, postres.
```

Comment:
Tendrías que trabajar esto en función de productos mínimamente viables. Esa "descripción" no veo que aporte nada.

Reply:
Entiendo.

En ese apartado mi intención era explicar ciertos requisitos que se deben de cumplir para que el sistema que propongo funcione.
Es decir, que alguien, dueño o administrador del sistema, debe incluir la información previa para que funcione.

Como esto no es ni Milestone ni Historia de Usuario, por eso lo había escrito ahí.

¿Cree que debería de incluirlo en otro lugar, de otra manera o carece de importancia en este caso?

Reply:
En la modificación del objetivo 0 pongo que evitéis que la gente tenga que describir nada. Si lo hacéis, no estáis empatizando con el cliente y no le aporta valor.